R/generateFilterValues.R

```diff
@@ -110,7 +109,11 @@ generateFilterValuesData = function(task, method = "randomForestSRC_importance",
     } else {
       stopf("You use more than 1 filter method. Please pass extra arguments via 'more.args' and not '...' to filter methods!")
     }
+  } else if (length(method) == 1L && length(more.args) > 1L) { # simple filter with 1 method and more.args
+    more.args = namedList(method, more.args)
   }
+  assertList(more.args, names = "unique", max.len = length(method))
+
```

Comment:
Please remove the unnecessary empty line.

Reply:
OK.